### DISCUSSION TITLE
How to apply custom zoom in a single story?

### DISCUSSION THREAD

**Discussion by simo54** [storybookjs/storybook#18212] [Help]
Without the action of `zoomin` and `zoomout` provided in the toolbar, is there a way to apply a custom zoom value for a story without using css? 

Standard zoom applied by default
![image](https://user-images.githubusercontent.com/65959785/168152464-19fef9ed-583c-4588-809b-9d05507d8d36.png)

I would like that when story is render the zoom value would be different 
![image](https://user-images.githubusercontent.com/65959785/168152804-3c47bbab-4838-4e9d-b00f-d50c2aefac13.png)

**Comment by bozzelliandrea** [comment]
Have you tried the `componentWrapperDecorator(story => '<div style="zoom: 4">${story}</div>')` ? With this build-in function you can manage your story style without modify any css property of the web component.
(but you need to add some css in the wrapper in this case).
